TESTING_WITH_IAF-TEST.md

```diff
@@ -66,7 +67,7 @@ See the Eclipse instructions for an important note about the `log.dir` setting.
 ## 4. Select database
 
 The ibis-adapterframeworkt-test project supports multiple databases. By default, an H2 local database is used.
-Docker projects for a number of other DMBSes are provided in iaf/docker/dbms. To use one of the provided databases, run the `rebuild.bat` script in the corresponding directory. (requires Docker to be installed on your machine). To configure the ibis-adapterframeworkt-test application, set in the catalina.properties or the VM Options the property `jdbc.dbms.default` to `oracle`, `mssql`, `mysql`, `mariadb` or `postgres`.
+Docker projects for a number of other DMBSes are provided in GitHub project https://github.com/ibissource/iaf-ci-images. To use one of the provided databases, run the `rebuild.bat` script in the corresponding directory. (requires Docker to be installed on your machine). To configure the ibis-adapterframeworkt-test application, set in the catalina.properties or the VM Options the property `jdbc.dbms.default` to `oracle`, `mssql`, `mysql`, `mariadb` or `postgres`. If you have no preference about the database you are working with, please do not choose Oracle. Working with Oracle databases is complicated because downloading the required drivers / docker images is complicated.
```

Comment:
Werken met Oracle is niet lastig, het creeren van een docker image voor Oracle is lastig.

Reply:
Verwerkt.

Reply:
Nog steeds: werken met Oracle databases is niet lastig. 
Zoals je het nu schrijft lijkt het net of het Framework niet goed samengaat met Oracle.